IssueTracker/IssueDb/dbo/StoredProcedures/Comment/spComment_GetALLByUserId.sql

```diff
@@ -2,8 +2,48 @@
 	@UserId UNIQUEIDENTIFIER
 AS
 begin
-	select * 
-	from dbo.[Comment]
-	where [Comment].IsDeleted = 0 /*FALSE*/ AND [Comment].UserId = @UserId
-	order by [Comment].Updated desc;
+
```

Comment:
You should have 2 selects here, one for comments and one for files, and you can map them into the aggregate with Dapper.